Amperfy/AppDelegate.swift

```diff
@@ -232,6 +240,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         configureNotificationHandling()
         initEventLogger()
         self.window = UIWindow(frame: UIScreen.main.bounds)
+        #if !targetEnvironment(macCatalyst)
+        let criteria = GCKDiscoveryCriteria(applicationID: kReceiverAppID)
+        let options = GCKCastOptions(discoveryCriteria: criteria)
+            GCKCastContext.setSharedInstanceWith(options)
+        #endif
```

Comment:
I would extract all this code into let say a `ChromeCastContiguration` class and avoid cluttering the AppDelegate.